### ISSUE TITLE
Enclosures: hidden action links lack names

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#540]
These links don't display, but they still exist. They don't have names, but they still show up in tab order.

We should probably not display them at all if they are not supposed to be used. If we need them to be there, they should not be focusable if they do not need to be used. If they must be focusable, they need to have names.

![enclosures-hidden-links](https://user-images.githubusercontent.com/45127691/102562102-56341480-40a4-11eb-80e3-999441f6e667.PNG)

**Comment by metamoni** [comment]
@CraigJZ Do you know why these actions are hidden? I'm looking at the Cohorts page and we have the same issue there - the Edit and Delete icons are there in the code, but they're not visible on the page.

**Comment by CraigJZ** [comment]
Hi @metamoni. Thanks for catching this. I believe this is unintentional and, interestingly, is happening in Chrome but not Firefox for me. It appears that the content adjusts to fit the table in Firefox but, in Chrome, those last 2 icons don't display unless other content is removed to make room. I will need to dig a little deeper into what might be causing this but will try to get an issue up for it soon. Thanks!
